.github/workflows/merge_develop.yml

```diff
@@ -0,0 +1,33 @@
+name: Merge develop to master
```

Comment:
Do we need a separate workflow for this? "Release stable" also forwards _master_.

Reply:
On a very few occasions, yes we do. `semantic-release` remembers the branch it was run in. So if we need to change something about the release, we must manually merge develop to master.